### ISSUE TITLE
A Properly documentation for Lambdas Integration.

### ISSUE THREAD

**Issue by lucaslacerdacl** [terraform-aws-modules/terraform-aws-eventbridge#29]
## Is your request related to a new offering from AWS?
NO

## Is your request related to a problem? Please describe.
NO

## Describe the solution you'd like.
A complete example for lambda integration.

## Describe alternatives you've considered.
Create an example for lambda.

## Additional context
Today we worked on integration with lambdas and found insufficient documentation in the repository about ways to integrate with serverless services. After a hard working day, we are currently stuck in how we could add constant (JSON text) in lambda targets. I need to know which property represents the info about the image attached.

![image](https://user-images.githubusercontent.com/6237330/140594193-6097b765-0b62-49b0-9731-1af1a040717b.png)

```hcl
module "eventbridge" {
  source = "terraform-aws-modules/eventbridge/aws"
  version = "1.12.0"

  create_bus = false
  
  rules = {
    my-service-cron = {
      description = "Lambda for serverless job"
      schedule_expression = "rate(5 minutes)"
    }
  }

  targets = {
    my-service-cron = [
      {
        name = "my-service-cron"
        arn  = "arn:aws:lambda:{AWS_ACCOUNT_IINFO}:{AWS_ACCOUNT_IINFO}:function:my-service-cron"
        ??????? -> which property is equivalent the image attached
      }
    ]
  }
}
```

**Comment by lucaslacerdacl** [comment]
https://github.com/terraform-aws-modules/terraform-aws-eventbridge/pull/28

**Comment by lucaslacerdacl** [comment]
Today I edit the property manually and run the plan command for debbuging.
Apparently what i needed is working:

```hcl
module "eventbridge" {
  source = "terraform-aws-modules/eventbridge/aws"
  version = "1.12.0"

  create_bus = false
  
  rules = {
    my-service-cron = {
      description = "Lambda for serverless job"
      schedule_expression = "rate(5 minutes)"
    }
  }

  targets = {
    my-service-cron = [
      {
        name = "my-service-cron"
        arn  = "arn:aws:lambda:{AWS_ACCOUNT_IINFO}:{AWS_ACCOUNT_IINFO}:function:my-service-cron"
        input = jsonencode({"job":"MY-SERVICE-CRON"})
      }
    ]
  }
}
```

**Comment by antonbabenko** [comment]
It's great that you found the solution. Let's work on #28 and it will be even better for the users who are coming after you.

**Comment by antonbabenko** [comment]
Fixed in #28. Thank you very much @lucaslacerdacl !
